Cargo.toml

```diff
@@ -1,6 +1,6 @@
 [package]
 name = "fastqc-rs"
-version = "0.3.1-alpha.0"
+version = "0.3.1-alpha.1"
```

Comment:
This will be automatically taken care of by me releasing a new version of the crate when this is merged. So it'd nice if you could revert this 😅

Reply:
I could not revert in the standard way but did revert back to .0

Reply:
No problem. The commits of the PR will be squashed anyways 😄